src/main/java/com/smartling/marketo/sdk/domain/email/DynamicContent.java

```diff
@@ -0,0 +1,33 @@
+package com.smartling.marketo.sdk.domain.email;
+
+import java.util.List;
+
+public class DynamicContent {
```

Comment:
Let's use lombok for getters/setters/constructors in DTO

Reply:
it's not used currently in this project. I don't see any reasons to spent time for this right now